zanata/zanata-platform
# feat(ZNTA-2302): Persist editor Difference toggle selection as a setting

JIRA issue URL: https://zanata.atlassian.net/browse/ZNTA-2302

The Editor Options task called for a settings checkbox to allow persisting selection of the Difference toggle option in the suggestions view.
We decided to instead persist the toggle option as the more user friendly option.

## Review
Moving over to using the settings reducer meant removing a lot of code, including some tests, of the suggestions reducer. The diff toggle option is now tested as a setting in the settings reducer.
## QA
The difference toggle selection in the alpha editors suggestions view should remain as it last was after refreshing the page.

## Checklist

- JIRA link
- Check target branch
- Make sure all commit statuses are green or otherwise documented reasons to ignore
- QA needs to evaluate against the JIRA ticket
- Changed files and commits make sense for this PR

See [Zanata Development Guidelines](https://github.com/zanata/zanata-platform/wiki/Development-Guidelines) more for information.

----
*This template can be updated in .github/PULL_REQUEST_TEMPLATE.md*


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/zanata/zanata-platform/670)
